2-mandatory/6-journey-planner.js

```diff
@@ -122,7 +132,11 @@ function getLocationName() {}
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+  let locationsStr = [];
+  locations.forEach((element) => {
+    element.includes(transportMode) ? locationsStr.push(element[0]) : null;
+  });
```

Comment:
This work well. Ideally, you should be making use of your getTransportModes method so that you are only checking the "transportMode" variable against the transport modes in the "locations" arrays. But you could equally argue that this is unnecessary!